orderbook/src/database/events.rs

```diff
@@ -150,7 +151,10 @@ async fn insert_trade(
                 .bind(event.order_uid.0.as_ref())
                 .bind(u256_to_big_decimal(&event.sell_amount_including_fee))
                 .bind(u256_to_big_decimal(&event.buy_amount))
-                .bind(u256_to_big_decimal(&event.fee_amount)),
+                .bind(u256_to_big_decimal(&event.fee_amount))
+                .bind(&event.tx_hash.as_bytes())
+                // TODO - get the settlement log index from transaction...
```

Comment:
This will be the tricky part 🙈

Reply:
So tricky in fact, that it is currently not possible since the older contracts do not emit a settlement event.

Reply:
Given that this PR is functional as is and can't be pushed beyond until we are using the most recent contracts, I might propose that we merge it (along with an `UPDATE trades SET settlement_index = 0` and maybe some dummy txHash) as is so that we can at least have txHash for the alpha release.